docs/go/how-to-set-workeroptions-in-go.md

```diff
@@ -269,7 +269,7 @@ The resolution is in seconds.
 ```go
 // ...
 workerOptions := worker.Options{
-	DisableStickyExecution: false,
+	StickyScheduleToStartTimeout: time.Second(5),
```

Comment:
Personally I would like to remove all of these single "options" sections for rarely set options. The API can detail advanced options, docs only need to detail important ones. Basically nobody sets this, it is a wasted docs page. Imagine if we had pages for every worker option in every SDK.

Reply:
Yeah; will check on this and remove if not required.